tests/RoboFile.ini.dist

```diff
@@ -8,7 +8,7 @@ cmsPath = joomla-cms3
 
 ; If you want to clone a different branch, you can set it here
 ;branch = staging
-branch = 3.9.6
+branch = 3.9.8
```

Comment:
@OscarIboy  just need to change value for 3.9.8 and don't need to change value `-` to `+` . thanks